configure.ac

```diff
@@ -156,6 +157,10 @@ AS_IF([test "x$enable_esapi" = xyes],
            AC_DEFINE([OSSL], [1], [OpenSSL cryptographic backend])
            TSS2_ESYS_CFLAGS_CRYPTO="$LIBCRYPTO_CFLAGS"
            TSS2_ESYS_LDFLAGS_CRYPTO="$LIBCRYPTO_LIBS"
+       ], [test "x$with_crypto" = xmbed], [
+           AC_DEFINE([MBED], [1], [mbedTLS cryptographic backend])
+           TSS2_ESYS_CFLAGS_CRYPTO="$LIBMBED_CFLAGS"
+           TSS2_ESYS_LDFLAGS_CRYPTO="-lmbedcrypto"
        ], AC_MSG_ERROR([Bad value for --with-crypto $with_crypto]))])
```

Comment:
If someone chooses "--with-crypto=mbed" then the configure script should also check if the mbedcrypto library is installed and fail gracefully if it isn't. Please add something like:
```
AC_CHECK_HEADER(mbedtls/md.h, [], [AC_MSG_ERROR([Please install mbed])])
```

Reply:
thank you for the suggestion, I added it with 50e50e6